packages/app-accounts/src/Creator.tsx

```diff
@@ -345,6 +354,16 @@ class Creator extends React.PureComponent<Props, State> {
       ? keyring.createAccountMnemonic(seed, password, { name })
       : keyring.createAccount(formatSeed(seed), password, { name });
 
+    try {
+      const json = pair.toJson(password);
```

Comment:
Is the `meta.whenCreated` timestamp property in the backup file needed somewhere? 
It's only being added `onCreateAccount()` / `saveAccount()` so it's not part of the `pair` object here. If it has to be included, I'd have to wait until the account has been created and then re-request it with keyring.getPair()

Reply:
It is mostly app-specified. At this point it really does not get used anywhere, although it could be useful at some point, i.e. we could quite feasibly show on the Account overview when the account was created and/or updated.

Reply:
Put another way - it is part of the metadata (i.e. `pair.meta`), so it can really be anything that we deem useful. (Or even the user does, i.e. at some point it would bvre great if they can add their own fields)

The only real part of meta that is currently visible to the user is the `name` property.

Reply:
Hmm... it's also the only property that is not generated on keyring.createAccountMnemonic/ keyring.createAccount but on keyring.saveAccount, which is called in both create functions.

Would it be ok to move the check from `saveAccount` to a separate method which then can be called from within the create functions?  I just pushed a suggestion. Like this, the `whenCreated` timestamp would be part of the initial `pair` object that is being returned.

Reply:
100%, makes sense. (Good to have eyes on that)